src/App.js

```diff
@@ -13,27 +13,22 @@ class App extends Component {
       error: '',
     }
   }
-  componentDidMount(event) {
-    fetchData
-      .getAllMovies()
+  componentDidMount() {
+    fetchData.getAllMovies()
       .then(data => {
         this.setState({moviesKey: data.movies})
       })
       .catch(error => this.setState({error: 'uh oh'}))
   }
 
   showMovie = id => {
-    // const movie = movieData['movies'].find((movie) => movie.id === id)
-
-    fetchData
-      .getOneMovie(id)
+    fetchData.getOneMovie(id)
```

Comment:
I'm seeing that you've got two distinct fetch calls - and I'm guessing the code in those fetches is pretty similar. I'd guess you could combine them into one fetch function in which you send through the whole url as an argument. That way you could use the same fetch for both calls. Let me know if you'd like me to elaborate!